homeassistant/components/avocent_dpdu/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "avocent_dpdu",
+  "name": "Avocent Direct PDU",
+  "codeowners": ["@wgprojects"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/avocent_dpdu",
+  "homekit": {},
+  "iot_class": "local_polling",
+  "requirements": ["avocentdpdu==1.1.0"],
```

Comment:
I don't have one of these units to test if it's intended behaviour, but here https://github.com/wgprojects/avocent_pdu/blob/f9703aed9b298d6f2991e2edac38f2e20dd359ed/src/avocentdpdu/avocentdpdu.py#L34 it appears that the string generated will be one digit too long. E.g. outlet index 1 on an 8-outlet PDU would generate `010000000` (9 digits).

Reply:
I can't believe I missed this. Good catch thanks.